rules/heading.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const rule = require('unified-lint-rule');
+const visit = require('unist-util-visit');
+const titleCase = require('to-title-case');
+
+module.exports = rule('remark-lint:awesome/heading', (ast, file) => {
+	let headings = 0;
+
+	visit(ast, (node, index) => {
+		if (node.type !== 'heading') {
+			return;
+		}
+
+		if (node.depth > 1) {
+			if (index !== 0) {
+				return;
+			}
+
+			file.message('Main list heading must be of depth 1', node);
+		}
+
+		for (const child of node.children) {
+			if (child.type !== 'text') {
+				continue;
+			}
+
+			const headingText = child.value;
+
+			if (headingText !== titleCase(headingText)) {
```

Comment:
my guess is this will introduce some false negatives. i think it'll be better to use a a case whitelist like we do with [list-item](https://github.com/sindresorhus/awesome-lint/blob/master/rules/list-item.js#L238)

Reply:
in particular, i think some list headers will likely be acronyms for capital case like `AWS`, for example.

Reply:
See de238e7faea153f7bd8050d2e3b3701f2d1d62da.